feature/experimental/gribi/ate_tests/route_addition_during_failover_test/README.md

```diff
@@ -16,22 +16,24 @@ Validate gRIBI route persistence during SSO
 
 *   Inject `1000` IPv4Entries(`IPBlock1: 198.18.196.1/22`) in default VRF with NHGID: `1`.
 
-*   Validate that the entries are installed as FIB_PROGRAMMED using getRPC.
+*   Validate that the entries are installed as FIB_PROGRAMMED using Get RPC.
 
-*   Send traffic from ATE port-1 to prefixes in IPBlock1 and ensure traffic flows 100% and reaches ATE port-2.
+*   Send traffic from ATE port-1 to prefixes in IPBlock1, ensure traffic flows 100% and reaches ATE port-2, stop the traffic.
 
 *   Start injecting another 1000 IPv4Entries(`IPBlock2: 198.18.100.1/22`) in default VRF with NHGID: #1. 
 
-*   Check for coredumps in the DUT and validate that none are present.
+*   Check for gRIBI coredumps in the DUT and validate that none are present.
```

Comment:
Can you help add a TODO here about checking not just gRIBI core-dumps? We prefer to check for any coredumps.